tabs/src/components/app/views/QueryRunner.jsx

```diff
@@ -5,7 +5,6 @@ import PropTypes from 'prop-types';
 import { QueryInput } from "./runner/QueryInput";
 import { Request } from "./runner/Request";
 import { Response } from "./runner/Response";
-import { useRangeKnob } from '@fluentui/docs-components';
```

Comment:
did we actually use a range knob? because if not the initial height value is 120px but you set it as just 200 (no units) in callGraph

Reply:
The reason is because the Monaco editor is not resizable